libensemble/libE.py

```diff
@@ -441,6 +441,24 @@ def libE_mpi_worker(libE_comm, sim_specs, gen_specs, libE_specs):
 # ==================== Local version ===============================
 
 
+def _retrieve_generator(gen_specs):
```

Comment:
I don't know why this is needed. If gen is on a thread, should not need to be pickled.

Reply:
I've removed on my branch for now.